doc/manual/gpu.tex

```diff
@@ -322,7 +323,7 @@ \subsection{Attributing Measurements to Source Code for NVIDIA GPUs}
 When HPCToolkit uses NVIDIA's CUPTI to monitor a GPU-accelerated application, 
 CUPTI notifies HPCToolkit every time it loads a CUDA binary, known as a CUBIN, into a GPU.
 At runtime, HPCToolkit computes a cryptographic hash of a CUBIN's contents and records the CUBIN into the execution's measurement directory. 
-For instance, if a GPU-accelerated application loaded CUBIN into a GPU, NVIDIA's CUPTI informed HPCToolkit that the CUBIN was being loaded, and HPCToolkit computed its cryptographic hash as {\tt 972349aed8}, then HPCToolkit would record {\tt 972349aed8.cubin} inside a {\tt cubins} subdirectory of an HPCToolkit measurement directory. 
+For instance, if a GPU-accelerated application loaded CUBIN into a GPU, NVIDIA's CUPTI informed HPCToolkit that the CUBIN was being loaded, and HPCToolkit computed its cryptographic hash as {\tt 972349aed8}, then HPCToolkit would record {\tt 972349aed8.cubin} inside a {\tt gpubins} subdirectory of an HPCToolkit measurement directory.
```

Comment:
the cubin will actually be recorded with the suffix .gpubin.

This brings up a good point. We should probably reorganize this chapter of the manual into an overview section that describes coarse and fine-grain measurements.  We now collect gpubin objects for all three types of GPUs.  

Do we need to run hpcstruct on the AMD gpu binaries? We MUST run them on Intel GPU binaries if we use GT-Pin or we get nothing.

Reply:
@jmellorcrummey 

> the cubin will actually be recorded with the suffix .gpubin.

Yes, I only change the directory name. 

> 
> This brings up a good point. We should probably reorganize this chapter of the manual into an overview section that describes coarse and fine-grain measurements. We now collect gpubin objects for all three types of GPUs.

Yes, I agree that such reogranization would be better. Right now, most information is under the CUDA subsection. 

> 
> Do we need to run hpcstruct on the AMD gpu binaries? We MUST run them on Intel GPU binaries if we use GT-Pin or we get nothing.

No, we do not need to run hpcstruct on AMD gpu binaries as AMD GPU binaries have standard DWARF for hpcprof to extract line info.